kb/licensing/how-is-mbedtls-protected.md

```diff
@@ -2,9 +2,9 @@
 
 ## Copyright
 
-Mbed TLS (formerly known as PolarSSL) is, like most software, protected by copyright. It is licensed for use under an open-source Apache 2.0 license. The *project* is owned by TrustedFirmware.org, and the code is owned by all the people and companies who have contributed to it, collectively known as *The Mbed TLS Contributors* (except for some files in the 3rdparty directory which have copyright as noted in the file headers).
+Mbed TLS (formerly known as PolarSSL) is, like most software, protected by copyright. It is licensed for use under a dual open-source Apache 2.0 OR GPL 2.0-or-later license. The *project* is owned by TrustedFirmware.org, and the code is owned by all the people and companies who have contributed to it, collectively known as *The Mbed TLS Contributors* (except for some files in the 3rdparty directory which have copyright as noted in the file headers).
```

Comment:
The part about 3rd party is at least missing a word or two (“have ?? copyright”), and could use a bit more information.
```suggestion
Mbed TLS (formerly known as PolarSSL) is, like most software, protected by copyright. It is licensed for use under a dual open-source Apache 2.0 OR GPL 2.0-or-later license. The *project* is owned by TrustedFirmware.org, and the code is owned by all the people and companies who have contributed to it, collectively known as *The Mbed TLS Contributors* (except for some files in the 3rdparty directory, not used in the default build, which have a different copyright as noted in the file headers, and only have the Apache 2.0 license).
```
This should perhaps be split to make the sentence less long.